package.json

```diff
@@ -2007,7 +2007,13 @@
           "default": true,
           "description": "%cmake-tools.configuration.cmake.loadCompileCommands.description%",
           "scope": "resource"
-        }
+        },
+		"cmake.automaticReconfigure": {
```

Comment:
@bobbrow we have "configureOnOpen" and "configureOnEdit", can we name this "configureOnSwitch" or something similar?

Reply:
I couldn't think of a `configureOn` name that would be able to describe both presets and Kits changes, so I went a different way with the suggestion that was more generic.

Reply:
So, at the moment we have two name proposals.
I will adjust the name once an agreement has been found.

Reply:
@bobbrow, can you please confirm that your resolving of the conversation means that the current name is ok?

Reply:
yes, it is ok